content/en/docs/concepts/architecture/nodes.md

```diff
@@ -255,7 +255,7 @@ The node controller checks the state of each node every `--node-monitor-period`
 Heartbeats, sent by Kubernetes nodes, help determine the availability of a node.
 
 There are two forms of heartbeats: updates of `NodeStatus` and the
-[Lease object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#lease-v1-coordination-k8s-io).
+{{< api-reference page="cluster-resources/lease-v1" >}} object.
```

Comment:
How are we gonna differentiate the following definitions if we use this page naming style?

- gv: `admissionregistration.k8s.io/v1`, kind: `ServiceReference`
- gv: `apiextensions.k8s.io/v1`, kind: `ServiceReference`
- gv: `apiregistration.k8s.io/v1`, kind: `ServiceReference`